src/inject/GPMInject/interface/voiceControls/SpeechRecognizer.js

```diff
@@ -95,7 +95,7 @@ export default class SpeechRecognizer {
 
     _.forEach(actions, (actionString) => {
       if (this._handlers[actionString.toLowerCase()]) {
-        console.error(`The action "${action.toLowerCase()}" already has a handler registered`); // eslint-disable-line
+        Logger.error(`The action "${action.toLowerCase()}" already has a handler registered`); // eslint-disable-line
```

Comment:
I would want this to go to the console aswell, not sure if that is how winston works or not